src/js/components/CreateServiceModalServicePickerOption.js

```diff
@@ -0,0 +1,48 @@
+import {Hooks} from 'PluginSDK';
+import React from 'react';
+
+import CreateServiceModalServicePickerOptionContent from './CreateServiceModalServicePickerOptionContent';
+import CreateServiceModalServicePickerOptionImage from './CreateServiceModalServicePickerOptionImage';
+import Panel from './Panel';
+
+function getChildOfType(children, type) {
+  return React.Children.toArray(children).filter(
+    (child) => child.type === type
+  );
+}
+
+function CreateServiceModalServicePickerOption({children, onOptionSelect}) {
+  const columnClass = Hooks.applyFilter(
```

Comment:
This is necessary to adjust the 3x1 grid into a 2x2 grid for this component, let me know if you can think of a better way to do it without hooks.